package.json

```diff
@@ -8,23 +8,24 @@
     "url": "https://github.com/apollographql/reason-apollo"
   },
   "scripts": {
-    "test:ci": "bsb -make-world",
-    "prepare": "npm link bs-platform"
+    "build": "bsb -make-world",
+    "start": "bsb -make-world -w",
+    "clean": "bsb -clean-world",
+    "test": "echo \"Error: no test specified\" && exit 1"
   },
   "peerDependencies": {
-    "apollo-cache-inmemory": "^1.0.0",
-    "apollo-link-http": "^1.0.0",
-    "apollo-client": "^2.0.4",
-    "bs-platform": "^2.0.0"
+    "apollo-cache-inmemory": "^1.1.5",
+    "apollo-link-http": "^1.3.2",
+    "apollo-client": "^2.2.0",
+    "bs-platform": "^2.1.0"
   },
-  "keywords": [
-    "Reason",
-    "Apollo",
-    "React",
-    "GraphQL"
-  ],
+  "keywords": ["Reason", "Apollo", "React", "GraphQL"],
   "author": "Grégoire Van der Auwermeulen <gregoirevandera@gmail.com>",
   "devDependencies": {
+    "apollo-cache-inmemory": "^1.1.5",
```

Comment:
The 4 devDependencies are also referenced in the peerDependencies

Reply:
I'd rather keep them all as peerDependencies

Reply:
Well, those are meant to be needed for the example files.

Actually, do you need to have the actual dependencies in `node_modules` installed if you write bindings? Does BS check those?

Reply:
No, you shouldn't have them in the node_modules. That's why I added them only as peerDependencies. When people install it, they need to have the dependency. That's the only requirement. I'm checking the other PR...